backend/config/services.yaml

```
@@ -17,6 +17,9 @@ parameters:
     # Define the directory where images will be uploaded
     uploads_directory: '%kernel.project_dir%/public/uploads/images' # Or your desired path
     
+    # Base URL for generating complete image URLs in production
+    app.base_url: '%env(default::APP_BASE_URL)%'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)